src/components/institution/SelectInstitutionSkeleton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Skeleton, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+export const SelectInstitutionSkeleton = () => {
+  const { t } = useTranslation();
+  return (
+    <Box sx={{ width: '100%' }}>
+      <Box sx={{ width: '100%' }}>
+        <Typography variant="h3" sx={{ fontWeight: 'normal' }}>
+          {t('common.select_institution')}
+        </Typography>
+        <Skeleton height={90} />
+      </Box>
+      <Box sx={{ width: '100%' }}>
+        <Typography variant="h3" sx={{ marginTop: '1rem', fontWeight: 'normal' }}>
+          {t('common.select_unit')}
+        </Typography>
+        <Skeleton height={90} />
+      </Box>
+    </Box>
```

Comment:
Burde ses i sammenheng med [kommentaren over](https://github.com/BIBSYSDEV/NVA-Frontend/pull/5758/files#r1642564743), men ser ut som en del av tags/styling her er overflødig da det bare er native <div>-s(?)
```suggestion
    <>
        <Typography variant="h3" sx={{ fontWeight: 'normal' }}>
          {t('common.select_institution')}
        </Typography>
        <Skeleton height={90} />
        <Typography variant="h3" sx={{ marginTop: '1rem', fontWeight: 'normal' }}>
          {t('common.select_unit')}
        </Typography>
        <Skeleton height={90} />
    </>
```